### PR TITLE
Enhancing class path (Allure plugins support)

### PR DESCRIPTION
**Pull request by volkovs**
- Minimal changes to support plugins
- Class path generation code is taken from Allure core
- Class path separator is taken from system property
- Related issue http://stackoverflow.com/questions/32252305/display-my-own-tab-using-command-line-tool/

### CODE REVIEW

**Comment by baev** [comment]
@volkovs thanks for contribution. 

As you know this repository was moved to allure-core. So I don't know should we release new version of this or should wait for new commandline from core.

Summon @eroshenkoam

**Comment by volkovs** [comment]
@baev We have checked cli tool from core some time ago and it didn't look mature (e.g. it didn't include all the jars from lib folder into class path). So we decided to keep using old one.

We don't mind waiting (we will use patched version), can't tell on behalf of others.

File: src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java [hunk 1]
```diff
@@ -89,4 +89,21 @@ protected String getLoggerConfigurationArgument() {
         return String.format("-Dorg.slf4j.simpleLogger.defaultLogLevel=%s",
                 isQuiet() || !isVerbose() ? "error" : "debug");
     }
+
+    private String getClassPathArgument() {
+        String version = getConfig().getCurrentVersion();
+        String separator = System.getProperty("path.separator");
+        Path configPath = getConfigPath().getParent().toAbsolutePath();
+        Path bundleJarPath = getExecutablePath(version);
+        String pluginsPath = getPluginsPath().toAbsolutePath() + "/*";
```
Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java]: windows? =(

Comment (volkovs) [inline src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java]: Do you see something I don't see?
Yes, tested on Windows. Windows understands `/` as path separators.

Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java]: okeey, I am a bit surprised that this works on windows - I thought that Windows executable should be `%s/bin/java.exe`

**Comment by baev** [comment]
@volkovs you can use `allure-commandline` from the core as well. This one is finally deprecated.

**Comment by volkovs** [comment]
@baev OK, thanks for letting us know

**Pull request closed without merging**